Makefile

```diff
@@ -100,4 +86,4 @@ kubectl-config:
 	# Makefiles run recipes in sub-processes. A sub-process cannot modify the parent process's environment.
 	# The best I (@EngHabu) can think of at the moment is to output this for the user to eval in the
 	# parent process.
-	echo "export KUBECONFIG=$(KUBECONFIG):~/.kube/config:$(KUBE_CONFIG)/k3s/k3s.yaml"
+	echo "export KUBECONFIG=~/.kube/config:$(FLYTE_DIR)/k3s/k3s.yaml"
```

Comment:
Can you explain this change? why are we dropping `$(KUBECONFIG)`?

Reply:
it's a mistake, BTW we don't need this because flytectl automatically setup env and ask the user to put them in their bash/zsh

Reply:
does it update `kubeconfig`? does it clear `kubeconfig` after `flytectl teardown` ?

Reply:
no it will not overnight any config